### PR TITLE
Add CCDB optional test

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

**Comment by awegrzyn** [comment]
In order to run the test locally set correct CCDB URL and:
```
-BOOST_AUTO_TEST_SUITE(optionalTest, *boost::unit_test::disabled())
+BOOST_AUTO_TEST_SUITE(optionalTest/*, *boost::unit_test::disabled()*/)
```

File: Framework/test/testCcdbDatabase.cxx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+{
+
+using namespace o2::quality_control::core;
+using namespace o2::quality_control::repository;
+
+const std::string CCDB_ENDPOINT = "ccdb-test.cern.ch:8080";
+std::unique_ptr<DatabaseInterface> BackendInstance;
+std::unordered_map<std::string, std::string> Objects;
+
+BOOST_AUTO_TEST_SUITE(optionalTest/*, *boost::unit_test::disabled()*/)
```
Comment (Barthelemy) [inline Framework/test/testCcdbDatabase.cxx]: Shouldn't it be uncommented by default ?

Comment (awegrzyn) [reply Framework/test/testCcdbDatabase.cxx]: yes, my mistake

**Comment by Barthelemy** [comment]
Test failures are unrelated.

**Pull request merged**